go/dbproxy/main.go

```diff
@@ -0,0 +1,141 @@
+package main
+
+import (
+	"almond-cloud/sql"
+	"errors"
+	"net/http"
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+	"gorm.io/gorm"
+)
+
+func main() {
+	dsn := "newuser:password@tcp(127.0.0.1:3306)/dbname?charset=utf8mb4&parseTime=True&loc=Local"
+	sql.InitMySQL(dsn)
+	r := gin.Default()
+
+	r.GET("/localtable/:name", localTableGetAll)
+	r.GET("/localtable/:name/:uniqueid/:userid", localTableGetOne)
+	r.DELETE("/localtable/:name/:uniqueid/:userid", localTableDeleteOne)
```

Comment:
Once we have authentication, we don't need :userId, correct?

Reply:
Yes, we will use auth token to map to the userid.